app/components/Icons/Confetti.tsx

```diff
@@ -0,0 +1,60 @@
+        fill="#F87171"
+        className={props.className}
+      />
+      <path
+        d="M52.5204 32.8855C53.2835 32.2089 54.3702 32.0373 55.3046 32.4459L56.0576 32.7751C57.038 33.2038 58.1803 32.7565 58.609 31.7761C59.0377 30.7956 58.5904 29.6533 57.61 29.2246L56.857 28.8954C54.5387 27.8817 51.8426 28.3075 49.9495 29.9862C49.1069 30.7334 47.8806 30.8576 46.9053 30.2945L46.3554 29.9771C45.4288 29.442 44.2438 29.7595 43.7088 30.6862C43.1737 31.6129 43.4913 32.7979 44.4179 33.3329L44.9678 33.6504C47.3876 35.0474 50.4299 34.7393 52.5204 32.8855Z"
+        fill="#F87171"
+        className={props.className}
+      />
+      <path
+        d="M36.112 10.4829C35.8117 10.5781 35.552 10.8378 35.0326 11.3572C34.5132 11.8766 34.2535 12.1363 34.1583 12.4365C34.0778 12.6905 34.0778 12.9632 34.1583 13.2172C34.2535 13.5175 34.5132 13.7772 35.0326 14.2966C35.552 14.816 35.8117 15.0757 36.112 15.1709C36.366 15.2514 36.6386 15.2514 36.8926 15.1709C37.1929 15.0757 37.4526 14.816 37.972 14.2966C38.4914 13.7772 38.7511 13.5175 38.8463 13.2172C38.9268 12.9632 38.9268 12.6905 38.8463 12.4365C38.7511 12.1363 38.4914 11.8766 37.972 11.3572C37.4526 10.8378 37.1929 10.5781 36.8926 10.4829C36.6386 10.4024 36.366 10.4024 36.112 10.4829Z"
+        fill="#F87171"
+        className={props.className}
+      />
+      <path
+        d="M50.293 19.2929C50.9269 18.659 51.2439 18.342 51.6015 18.2043C52.0199 18.043 52.4834 18.043 52.9018 18.2043C53.2594 18.342 53.5763 18.659 54.2103 19.2929C54.8442 19.9268 55.1611 20.2438 55.2989 20.6014C55.4601 21.0198 55.4601 21.4832 55.2989 21.9017C55.1611 22.2593 54.8442 22.5762 54.2103 23.2101C53.5763 23.8441 53.2594 24.161 52.9018 24.2988C52.4834 24.46 52.0199 24.46 51.6015 24.2988C51.2439 24.161 50.9269 23.8441 50.293 23.2101C49.6591 22.5762 49.3421 22.2593 49.2044 21.9017C49.0431 21.4832 49.0431 21.0198 49.2044 20.6014C49.3421 20.2438 49.6591 19.9268 50.293 19.2929Z"
+        fill="#F87171"
+        className={props.className}
+      />
+      <path
+        d="M19.8554 10.1804C19.3143 9.63926 18.4369 9.63926 17.8958 10.1804C17.3547 10.7215 17.3547 11.5989 17.8958 12.14C18.4369 12.6811 19.3143 12.6811 19.8554 12.14C20.3965 11.5989 20.3965 10.7215 19.8554 10.1804Z"
+        fill="#F87171"
+        className={props.className}
+      />
+      <path
+        d="M49.2339 39.5595C49.775 39.0184 50.6523 39.0184 51.1935 39.5595C51.7346 40.1006 51.7346 40.978 51.1935 41.5191C50.6523 42.0602 49.775 42.0602 49.2339 41.5191C48.6927 40.978 48.6927 40.1006 49.2339 39.5595Z"
+        fill="#F87171"
+        className={props.className}
+      />
+      <path
+        d="M47.168 25.1654C46.6269 24.6243 45.7496 24.6243 45.2084 25.1654C44.6673 25.7065 44.6673 26.5839 45.2084 27.125C45.7496 27.6661 46.6269 27.6661 47.168 27.125C47.7092 26.5839 47.7092 25.7065 47.168 25.1654Z"
+        fill="#F87171"
```

Comment:
suggestion: instead of having the hardcoded color, you could use fill="currentColor" that will inherit the text color of its parent or if not change to ` fill=props.fill ?? "#F87171"` so it can be reused on the future